readme.md

```diff
@@ -81,7 +95,12 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+var geraldo = (name: "Geraldo", id: 1)
```

Comment:
this bit of code would error out. Because its just key value pairs inside of parens. You could do something like this though `Student.find(1)`(assuming you know geraldos id is 1) or `Student.find_by(name: "Geraldo"). The rest of your code would work great given this instantiation of `geraldo` is correct.